pkg/vex/vex.go

```diff
@@ -123,7 +124,7 @@ func (vexDoc *VEX) StatementFromID(id string) *Statement {
 // artifact as it changes over time.
 func Sort(docs []*VEX) []*VEX {
 	sort.Slice(docs, func(i, j int) bool {
-		return docs[i].Timestamp.Before(docs[j].Timestamp)
+		return docs[i].Timestamp.Before(*(docs[j].Timestamp))
```

Comment:
We have to be careful about a nil pointer dereference. Maybe there should be a helper method on `vex.VEX` to return its timestamp, which will either be `doc.Timestamp` or it should be the latest value among the statements? Otherwise it's an error condition